wasp/src/main/java/com/orhanobut/wasp/Wasp.java

```diff
@@ -189,6 +189,9 @@ public Wasp build() {
         }
 
         private void init() {
+            if (endPointUrl == null) {
```

Comment:
without endPoint nothing would work. It is ok to throw NPE here in my opinion as we do for other things.